### ISSUE TITLE
Ball website image missing alt text

### ISSUE THREAD

**Issue by LikkanChung** [CSSUoB/cssuob.github.io#407]
![image](https://user-images.githubusercontent.com/20641536/232144071-7edb5b53-9eb6-49fe-b15f-199f449e1109.png)

**Comment by LikkanChung** [comment]
As a User of the CSS Ball website, I would like all images to have alt-text, such that I am able to use it accessibly. 

AC: 
* All images on the webpage have alt-text
